include/quick_inventory

```diff
@@ -0,0 +1,93 @@
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+###############################################################################
+
+# Probably the easier and quickest way to get a resouce inventory in AWS.
+
+# Lists (almost) all resources in all regions based on:
+# https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/API_GetResources.html
+
+# this list is not accurate https://docs.aws.amazon.com/ARG/latest/userguide/supported-resources.html
+# don't fully trust that list column "Tag-based Groups". 
+# as far as I can see it doesn't include:
+# CodeBuild build projects like arn:aws:codebuild:eu-west-1:123456789012:build/Abcdefg
+# despite the supported-resources documentation page
+
+# Requires IAM permissions:
+# resource-groups:Get*
+# resource-groups:GroupResources
+
+
+# Quick inventory part
+quick_inventory(){
+    # Create a temporary file to store all resources arn 
+    TEMP_INVENTORY_FILE=$(mktemp -t prowler-inventory-${ACCOUNT_NUM}.XXXXXXXXXX)
+    INVENTORY_FILE="$OUTPUT_DIR/prowler-inventory-${ACCOUNT_NUM}-${OUTPUT_DATE}.csv"
+
+    echo "-=- Running the inventory for AWS Account ${OK}${ACCOUNT_NUM}${NORMAL} -=-"
+    echo -e "\n # Counting resources by region:"
+    # get all resources per region and getting only their ARN
+    for region in $REGIONS; do
+        echo -ne " - $region: "
+        AWS_RESOURCES=$($AWSCLI resourcegroupstaggingapi get-resources \
+            $PROFILE_OPT \
+            --region $region \
+            --query ResourceTagMappingList[].ResourceARN \
+            --output json 2>&1)
+        if [[ $(echo "$AWS_RESOURCES" | grep -E 'service control policy') ]]; then
+            echo -e "${BAD}blocked by a SCP${NORMAL}"
+            continue
+        fi
+
+        if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "$AWS_RESOURCES"; then
+            echo "${BAD}Access Denied trying to get AWS Resources${NORMAL}"
+            continue
+        fi
+        jq -r '.[]' <<< "${AWS_RESOURCES}" >> "${TEMP_INVENTORY_FILE}" 
+        TOTAL_RESOURCES_FOUND_REGION=$(grep -c "${region}" "${TEMP_INVENTORY_FILE}")
+
+        echo -e "${OK}${TOTAL_RESOURCES_FOUND_REGION}${NORMAL} resources!"
+        
+        TOTAL_RESOURCES_FOUND=$(("${TOTAL_RESOURCES_FOUND}"+"${TOTAL_RESOURCES_FOUND_REGION}"))
+    done
+    echo -e "Total resources found: ${OK}$TOTAL_RESOURCES_FOUND${NORMAL}\n"
+
+    # Generate file in CSV format
+    # send header first
+    echo "aws_partition,aws_service,aws_region,aws_account_id,subservice_resource,arn" > "${INVENTORY_FILE}"
+    
+    while IFS=: read -r arn aws_partition aws_service aws_region aws_account_id subservice_resource; do
```

Comment:
Why do you need `arn` variable?

Reply:
to give it to the user for possible automation, why not?